package.json

```diff
@@ -40,6 +43,10 @@
     "sinon": "^2.1.0",
     "sinon-chai": "^2.9.0",
     "style-loader": "^0.14.1",
-    "webpack": "^2.2.1"
+    "webpack": "^2.2.1",
+    "webpack-node-externals": "^1.5.4"
+  },
+  "dependencies": {
+    "hpq": "^1.0.1"
```

Comment:
>Minor but do you think we should move it inside the project or something?

Inside in what way? This seems like a handy tool regardless of WordPress context, hence my initial desire to built is separate. Or do you mean more like how we're treating React with script dependencies?